docs/development.rst

```diff
@@ -1,582 +1,93 @@
+    cd test_server
+    source bin/activate
+    pip install -U pip
+    pip install -U setuptools
 
-#. Create a PR against ``master`` that has the following changes: 
 
-   #. update the release notes in ``docs/status.rst`` with a description of what is in the release 
-   #. modify ``requirements.txt`` to pin the ga4gh packages to specific versions 
-   #. modify ``constraints.txt`` to comment out all the lines referencing ga4gh packages
-   #. modify ``constraints.txt.default`` to have the same contents as ``constraints.txt``
-   #. modify ``docs/environment.yml`` to pin the ga4gh packages to specific versions
+    pip install candig-server
+    pip install candig-ingest==1.3.1
 
-#. Once this has been merged, tag the release on GitHub (on the `releases
-   <https://github.com/ga4gh/ga4gh-server/releases>`_ page) with the
-   appropriate version number.
-#. Fetch the tag from the upstream repo, and checkout this tag.
-#. Create the distribution tarball using ``python setup.py sdist``, and then
-   upload the resulting tarball to PyPI using 
-   ``twine upload dist/ga4gh-$MAJOR.$MINOR.$PATCH.tar.gz`` (using 
-   the correct file name).
-#. Verify that the documentation at
-   http://ga4gh-server.readthedocs.org/en/stable/
-   is for the correct version (it may take a few minutes for this to
-   happen after the release has been tagged on GitHub).  The release
-   notes docs should have changed, so that is a good section to look at
-   to confirm the change.
+    mkdir candig-example-data
 
-All of the above steps after the tag is dropped on the target commit are now
-`automated <https://docs.travis-ci.com/user/deployment/pypi/>`_ using
-Travis' capability to deploy to Pypi.
+    wget https://raw.githubusercontent.com/CanDIG/candig-ingest/master/candig/ingest/mock_data/clinical_metadata_tier1.json
+    wget https://raw.githubusercontent.com/CanDIG/candig-ingest/master/candig/ingest/mock_data/clinical_metadata_tier2.json
+    ingest candig-example-data/registry.db mock1 clinical_metadata_tier1.json
+    ingest candig-example-data/registry.db mock2 clinical_metadata_tier2.json
 
-Note that once a filename is uploaded to pypi, it can not be reuploaded (`see this pypi issue <https://github.com/pypa/packaging-problems/issues/74>`_), so if a mistake is made in uploading a particular tag, a new tag will need to be created to perform the upload.  Usually this means revving the ``post`` number at the end of the tag.
+    candig_repo add-peer candig-example-data/registry.db http://0.0.0.0:3001
 
-+++++++++++++++++++++
-Stable bugfix release
-+++++++++++++++++++++
+    echo "test server set up has completed."
 
-When a minor version series has ended because of some significant shift
-in the server internals, there will be a period when the ``master`` branch is not
-in a releasable state. If a bugfix release is required during this period,
-we create a release using the following process:
+    candig_server --host 0.0.0.0 --port 3000
```

Comment:
A registered peer, but only the main server holds data. Is it to show the process to add peers and to get back more meaningful results?

Reply:
this line? `candig_repo add-peer candig-example-data/registry.db http://0.0.0.0:3001`?

Reply:
I am asking more in general, a server on port 3001 is not deployed just got registered

Reply:
yes, this could be confusing, i will rewrite the corresponding parts